src/main.rs

```diff
@@ -73,6 +79,9 @@ type SpiResult<T> = Result<T, nb::Error<core::convert::Infallible>>;
 type EnabledUart = hal::uart::UartPeripheral<rp2040_hal::uart::Enabled, pac::UART0,
     (rp2040_hal::gpio::Pin<Gpio0, rp2040_hal::gpio::Function<rp2040_hal::gpio::Uart>>, rp2040_hal::gpio::Pin<Gpio1, rp2040_hal::gpio::Function<rp2040_hal::gpio::Uart>>)>;
 
+//type Params = Vec::<u8, PARAMS_ARRAY_LEN>;
```

Comment:
```suggestion
```

Reply:
Resolved in [following commit](https://github.com/Jim-Hodapp-Coaching/esp32-pico-wifi/pull/4/commits/c10bb540c941bdb44dbff93ce28af3736f1ca7b1)